modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,104 @@
+
+For the HyperV platform, Ignition will read its configuration file from the HyperV hypervisor in parts and then reassemble it.  Before this can occur, you can use `kvpctl` and its _add-ign_ subcommand to split your Ignition file and assign it to the virtual machine.  The syntax for the command is as follows:
+
+[source, powershell]
+----
+.\kvpctl.exe <name_of_vm> add-ign <path_to_ign_file> <ignition_config_prefix>
+----
+
+When Ignition is run on first boot, it will look for a pre-defined key name and reassemble the Ignition configuration file.  The pre-defined key name is *ignition.config.*.  Be sure to note the trailing `.` on the prefix key name.
+
+For example:
+
+[source, powershell]
+----
+.\kvpctl.exe podman-machine-default add-ign C:\Users\joe\myvm.ign ignition.config.
+added key:  ignition.config.0
+added key:  ignition.config.1
+added key:  ignition.config.2
+added key:  ignition.config.3
+added key:  ignition.config.4
+added key:  ignition.config.5
+added key:  ignition.config.6
+----
+
+
+=== Starting the VM
+
+Before starting the VM, be certain you have completed the step xref:_setting_the_ignition_config[ Setting the Ignition config].
+
+There are several ways to start a virtual machine depending on the context of the HyperV Manager graphical interface.  To start a virtual machine, for example,  you can right-click on the virtual machine and select _Start_.
+
+=== Viewing the key value pairs assigned to your virtual machine
+
+You can view the key value pairs that are assigned to your machine with `kvpctl` and its _get_ subcommand. You may only get key value pairs when the virtual machine is _running_.  This is by design from Microsoft.
+
+[source, powershell]
+----
+.\kvpctl.exe <name_of_vm> get
+----
+
+For example:
+
+[source, bash]
+----
+> .\kvpctl.exe myvm get
+ignition.config.3 = th":"/etc/containers/registries.conf..."
+ignition.config.4 = ,"contents":{"source":"data:,makeste..."
+ignition.config.5 = nabled":false,"mask":true,"name":"do..."
+ignition.config.6 = service\n\n[Service]\nExecStart=/usr..."
+ignition.config.0 = {"ignition":{"config":{"replace":{"v..."
+ignition.config.1 = default.target.wants","user":{"name"..."
+ignition.config.2 = "user":{"name":"root"},"contents":{"..."
+----
+
+=== Removing the Ignition config
+
+When removing your virtual machine, you should also remove the keys you added for your virtual machine.  This can also be done with the `kvpctl` utility.  You can remove keys while the virtual is running or stopped.
+
+[source, bash]
+----
+> .\kvpctl.exe <name_of_vm> rm [ignition_key_names]
+----
+
+For example:
+
+[source, bash]
+----
+> .\kvpctl.exe podman-machine-default rm ignition.config.0 ignition.config.1 ignition.config.2 ignition.config.3 ignition.config.4 ignition.config.5 ignition.config.6
```

Comment:
```suggestion
[source, powershell]
----
.\kvpctl.exe podman-machine-default rm ignition.config.0 ignition.config.1 ignition.config.2 ignition.config.3 ignition.config.4 ignition.config.5 ignition.config.6
```

According to the [docs](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-highlighter/#shell-vs-console) a `source` block shouldn't include the prompt if declared as a specific shell language.

Also the example should probably be based around Powershell to facilitate using Windows native tooling (as implied by the `.exe`).